resources/core/charts/logging/templates/tests/logging.yaml

```diff
@@ -0,0 +1,62 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: test-{{ template "fullname" . }}
+  labels:
+    helm-chart-test: "true"
+rules:
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["create", "delete", "get", "list"]
```

Comment:
can you reduce this privileges to only create/delete pods with specific name and namespace?

Reply:
added resourceNames level privileges